test/functional/rpc_fundrawtransaction.py

```diff
@@ -305,6 +318,7 @@ def test_two_vin(self):
         ###########################################
         # test a fundrawtransaction with two VINs #
```

Comment:
f.e. just convert this line into a print statement

Reply:
Jup, can remove this now

Reply:
Done, ASCII art blocks removed.